src/main/java/org/icgc/argo/workflow/relay/service/SplitterService.java

```diff
@@ -119,7 +119,7 @@ public void split(JsonNode event) {
     } else if (event.has("workflowUrl")
         && event.has("runId")
         && event.has("event")
-        && event.has("utcTime")) {
+        && event.has("utcTime")) { // These are the QUEUED and INITIALIZING events --->  "runId":"wes-9f2ede5f8dfd41e2a68309ecfa7a1f5c","event":"INITIALIZING","utcTime":"2023-07-26T14:22:02Z","workflowUrl":......
```

Comment:
I think this comment needs a more prosaic/detailed format, if you meant to leave it here